Edits/azure-service-bus-management-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+
+To configure the global element for Azure Service Bus Management Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, configure Shared Access Signature (SAS) authentication by entering values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Service Namespace |Namespace for the Azure Service Bus Service within your subscription
+|Shared Access Key Name | Name of your SAS token
+|Shared Access Key | SAS token
+|===
++
+The following screenshot shows an example of configuring Shared Access Signature (SAS) authentication. This screenshot uses property placeholders in place of field values. For information on using property placeholders, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
++
+.SAS authentication fields
+image::azure-sb-mgmt-authentication.png[To configure SAS authentication, complete the fields on the *General* tab.]
++
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[configure-other-fields]]
+== Configure the Other Connector Fields
+
+After you configure a global element for Azure Service Bus Management Connector, configure the other required fields for the connector operation.
+
+For example, to create or update a queue or topic, use the `Create or Update Entity` operation and complete these fields:
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Entity (Queue or Topic) | Enter the name of the entity to create or update.
+|If Match |
+To update an existing entity, set this header to `*`. Otherwise, leave this field blank.
+|Content | Enter the queue description or topic.
+|===
+
+[[view-app-log]]
+== View the App Log
+
+You can view the app log as follows:
+
+* If you’re running the app from the Anypoint Platform, the output is visible in the Anypoint Studio console window.
+* If you’re running the app using Mule from the command line, the app log is visible in your OS console.
+Unless the log file path was customized in the app’s log file (`log4j2.xml`), you can also view the app log in this default location:
+`MULE_HOME/logs/<app-name>.log`
+For more information about the app log, see xref:mule-runtime::logging-in-mule.adoc[Configuring Logging].
+
+== Next Step
+
+See xref:azure-service-bus-management-connector-config-topics.adoc[Additional Configuration Information] for more configuration information.
+
```

Comment:
Yeah...I really think we should not have to send them to a whole new topic just to describe this one impediment. That info belongs in the global element config info here.